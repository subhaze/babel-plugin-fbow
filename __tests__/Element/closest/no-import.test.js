const babel = require("babel-core");
const plugin = require("../../../index.js");

const testCode = `
const a = document.querySelector("a");
const div = document.querySelector("div");
a.closest("div");
div.closest("p");
`;

it("doesn't add closest import", () => {
	const { code } = babel.transform(testCode, {
		plugins: [plugin],
		sourceType: "script",
	});
	expect(code).toMatchSnapshot();
});
