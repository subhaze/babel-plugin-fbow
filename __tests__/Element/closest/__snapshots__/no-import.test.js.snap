// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`doesn't add closest import 1`] = `
"
const a = document.querySelector(\\"a\\");
const div = document.querySelector(\\"div\\");
a.closest(\\"div\\");
div.closest(\\"p\\");"
`;
